### ISSUE TITLE
lotus-miner window post exit PoSt cycle ERROR

### ISSUE THREAD

**Issue by yangyueqiangyy** [filecoin-project/lotus#10638]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [X] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus-miner version 1.20.0+mainnet+git.33bec58be
```

### Repro Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

### Describe the Bug

When lotus-miner occurs ’RPCConnectionError‘, Window post will exit PoSt cycle, even the lotus daemon restore normal.
When window post exit PoSt cycle, lotus-miner no longer do the window post.It is restore normal, when restart lotus-miner process.

### Logging Information

```text
2023-04-09T05:16:00.187+0800    INFO    wdpost  wdpost/wdpost_run.go:260        starting PoSt cycle     {"cycle": "2023-04-09T05:16:00.186+0800", "manual": false, "ts": "[bafy2bzacecwxda7fm2zhyxijaohwehcybsunf5n2b3x562z66ifupm2tmzvw4]", "deadline": 34}
2023-04-09T05:16:02.664+0800    INFO    wdpost  wdpost/wdpost_run.go:262        post cycle done {"cycle": "2023-04-09T05:16:00.186+0800", "took": 2.477742559}
2023-04-09T05:44:32.409+0800    WARN    rpc     go-jsonrpc@v0.2.1/websocket.go:763      websocket long time no response {"lastAction": "incoming", "time": 30.00045346}
2023-04-09T05:44:32.409+0800    INFO    retry   retry/retry.go:17       Retrying after error:RPCConnectionError
2023-04-09T05:44:32.409+0800    WARN    wdpost  wdpost/wdpost_sched.go:171      window post scheduler notifs channel closed
2023-04-09T05:44:34.184+0800    WARN    events  events/observer.go:61   listenHeadChanges quit
2023-04-09T05:44:35.184+0800    INFO    events  events/observer.go:70   restarting listenHeadChanges
2023-04-09T07:32:15.126+0800    WARN    rpc     go-jsonrpc@v0.2.1/websocket.go:763      websocket long time no response {"lastAction": "incoming(xrpc.ch.val,<nil>)", "time": 13.595591595}
2023-04-09T11:47:01.164+0800    WARN    rpc     go-jsonrpc@v0.2.1/client.go:392 rpc output message buffer       {"n": 11}
```

**Comment by rjan90** [comment]
Hey @yangyueqiangyy! The issue with the window post shceduler notifs channel closed has been fixed with this PR: https://github.com/filecoin-project/lotus/pull/10395. It is is coming with the v1.21.0 release

**Comment by yangyueqiangyy** [comment]
> Hey @yangyueqiangyy! The issue with the window post shceduler notifs channel closed has been fixed with this PR: #10395. It is is coming with the v1.21.0 release
Thank you !
